strongbox-web-core/src/main/java/org/carlspring/strongbox/forms/RoleForm.java

```diff
@@ -0,0 +1,70 @@
+package org.carlspring.strongbox.forms;
+
+import org.carlspring.strongbox.validation.UniqueRoleName;
+
+import javax.validation.constraints.NotEmpty;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.google.common.collect.Sets;
+
+/**
+ * @author Pablo Tirado
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RoleForm
+{
+
+    @NotEmpty(message = "This field must not be empty.")
```

Comment:
Same as my earlier comment about the `name` field.

Reply:
I agree too.